web/src/fold_dict.ts

```diff
@@ -48,7 +48,7 @@ export class FoldDict<V> {
         }
     }
 
-    *values(): Iterator<V> {
+    *values(): IterableIterator<V> {
```

Comment:
Why this change -- I thought we made `BinaryDict` no longer use `FoldDict`? Maybe worth putting in a separate commit if it's a bug fix.